docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -378,6 +378,66 @@ For other regular expressions, the code is the same as for the ‘match’ funct
 
 The same thing as ‘like’, but negative.
 
+## iLike {#ilike}
+
+Case insensitive variant of [like](https://clickhouse.tech/docs/en/sql-reference/functions/string-search-functions/#function-like) function and works the same as the `ILIKE` operator.
```

Comment:
The function doesn't work like `ILIKE`. `ILIKE` implementation is based on this function. You can write that the user can run ILIKE operator instead of the function.